libc/test/src/math/performance_testing/CMakeLists.txt

```diff
@@ -164,6 +164,17 @@ add_perf_binary(
     -fno-builtin
 )
 
+add_perf_binary(
+  exp2f16_perf
+  SRCS
+    exp2f16_perf.cpp
+  DEPENDS
+    .single_input_single_output_diff
+    libc.src.math.exp2f16
+  COMPILE_OPTIONS
+    -fno-builtin
```

Comment:
This shouldn't change anything yet. I added it now so that we don't forget it when the day comes that we remove the `placeholderf16` function and benchmark against the system libc's function.

Reply:
What's this testing exactly? And if we want to disable a single builtin it's better to just do `-fno-builtin-exp2f16` if that's what it's called.

Reply:
All the math performance tests, except one, benchmark LLVM libc functions against respective system libc functions, so all of them except that one have `-fno-builtin`. I don't know why fine-grained `-fno-builtin-<function>` flags weren't used.

In this case, since the system libc shouldn't have `float16` math functions yet, I benchmark against a placeholder function for now. It would probably be better if at some point we just rewrote our performance testing code to use Google Benchmark, and also allow us to benchmark a single function instead of comparing two, while we're at it.